### ISSUE TITLE
Error add light after upgrading to 2.5

### ISSUE THREAD

**Issue by mainTAP** [timniklas/hass-govee_light_ble#58]
I've upgraded to version 2.5 but now I cannot re-add my H6181 LED strip , it returns "Unknown error occurred" in the GUI, and this is what I see in the logs :

```
Logger: homeassistant.config_entries
Source: config_entries.py:640
First occurred: 14:52:09 (3 occurrences)
Last logged: 14:57:35

Error setting up entry ihoment_H6181_7488 for govee_light_ble
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 640, in __async_setup_with_context
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/govee_light_ble/__init__.py", line 45, in async_setup_entry
    await coordinator.async_config_entry_first_refresh()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 306, in async_config_entry_first_refresh
    await self._async_refresh(
        log_failures=False, raise_on_auth_failed=True, raise_on_entry_error=True
    )
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 379, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/govee_light_ble/coordinator.py", line 71, in _async_update_data
    await self._api.sendPacketBuffer()
  File "/config/custom_components/govee_light_ble/api.py", line 134, in sendPacketBuffer
    await self.stop_event.wait()
  File "/usr/local/lib/python3.13/asyncio/locks.py", line 213, in wait
    await fut
asyncio.exceptions.CancelledError
```

Thank you

**Comment by vladislabv** [comment]
Same here, happened to me with H613D_AEEE, the LED strip can't be initialized. However v2.4.1 works fine.

**Comment by rorydcampbell** [comment]
++

**Comment by timniklas** [comment]
Is this still happening in v2.6-pr1 ?

**Comment by vladislabv** [comment]
@timniklas Legacy. Yes, still the same error.

**Comment by timniklas** [comment]
> @timniklas Legacy. Yes, still the same error.

Can you give some more error information? At least he causing line should have changed with ne new version

**Comment by mainTAP** [comment]
In the 2.6 pr1 , the device seems to be stuck in "Initializing" and I cannot even delete it to try to re-add it.
Looked at the logs I see this :

```
This error originated from a custom integration.

Logger: custom_components.govee_light_ble.coordinator
Source: helpers/update_coordinator.py:379
integration: GOVEE Bluetooth Lights (documentation, issues)
First occurred: 19:18:02 (1 occurrences)
Last logged: 19:18:02

Unexpected error fetching govee_light_ble (A4:C1:38:7C:74:88) data
Traceback (most recent call last):
  File "/config/custom_components/govee_light_ble/api.py", line 136, in sendPacketBuffer
    await self.stop_event.wait()
  File "/usr/local/lib/python3.13/asyncio/locks.py", line 213, in wait
    await fut
asyncio.exceptions.CancelledError: Global task timeout

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 379, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/govee_light_ble/coordinator.py", line 71, in _async_update_data
    await self._api.sendPacketBuffer()
  File "/config/custom_components/govee_light_ble/api.py", line 140, in sendPacketBuffer
    await self._stopReceiving()
  File "/config/custom_components/govee_light_ble/api.py", line 120, in _stopReceiving
    await self._client.stop_notify(READ_CHARACTERISTIC_UUID)
  File "/usr/local/lib/python3.13/site-packages/bleak/__init__.py", line 862, in stop_notify
    await self._backend.stop_notify(char_specifier)
  File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 78, in _async_wrap_bluetooth_operation
    return await func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 722, in stop_notify
    self._raise_if_not_connected()
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
  File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 733, in _raise_if_not_connected
    raise BleakError(f"{self._description} is not connected")
bleak.exc.BleakError: esp32-bluetooth-proxy-d94080 [08:D1:F9:D9:40:80]: ihoment_H6181_7488 - A4:C1:38:7C:74:88 is not connected
```

**Comment by timniklas** [comment]
> In the 2.6 pr1 , the device seems to be stuck in "Initializing" and I cannot even delete it to try to re-add it. Looked at the logs I see this :
> 
> ```
> This error originated from a custom integration.
> 
> Logger: custom_components.govee_light_ble.coordinator
> Source: helpers/update_coordinator.py:379
> integration: GOVEE Bluetooth Lights (documentation, issues)
> First occurred: 19:18:02 (1 occurrences)
> Last logged: 19:18:02
> 
> Unexpected error fetching govee_light_ble (A4:C1:38:7C:74:88) data
> Traceback (most recent call last):
>   File "/config/custom_components/govee_light_ble/api.py", line 136, in sendPacketBuffer
>     await self.stop_event.wait()
>   File "/usr/local/lib/python3.13/asyncio/locks.py", line 213, in wait
>     await fut
> asyncio.exceptions.CancelledError: Global task timeout
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 379, in _async_refresh
>     self.data = await self._async_update_data()
>                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/config/custom_components/govee_light_ble/coordinator.py", line 71, in _async_update_data
>     await self._api.sendPacketBuffer()
>   File "/config/custom_components/govee_light_ble/api.py", line 140, in sendPacketBuffer
>     await self._stopReceiving()
>   File "/config/custom_components/govee_light_ble/api.py", line 120, in _stopReceiving
>     await self._client.stop_notify(READ_CHARACTERISTIC_UUID)
>   File "/usr/local/lib/python3.13/site-packages/bleak/__init__.py", line 862, in stop_notify
>     await self._backend.stop_notify(char_specifier)
>   File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 78, in _async_wrap_bluetooth_operation
>     return await func(self, *args, **kwargs)
>            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>   File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 722, in stop_notify
>     self._raise_if_not_connected()
>     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^
>   File "/usr/local/lib/python3.13/site-packages/bleak_esphome/backend/client.py", line 733, in _raise_if_not_connected
>     raise BleakError(f"{self._description} is not connected")
> bleak.exc.BleakError: esp32-bluetooth-proxy-d94080 [08:D1:F9:D9:40:80]: ihoment_H6181_7488 - A4:C1:38:7C:74:88 is not connected
> ```

This seems to be an error related to your Esphome proxy, could you try version 2.6-pre2 please?

**Comment by timniklas** [comment]
The Initializing takes some time, maybe 15 seconds per device. Thats to be exptected, since die Govee devices take a log time to connect.

**Comment by timniklas** [comment]
should be fixed with v2.6.1

**Comment by mainTAP** [comment]
Thank you very much, it's working in 2.6.1
